File: app/src/main/java/com/example/findmyrahmah/Restaurant.java
package com.example.findmyrahmah;

public class Restaurant {
    private final String restaurantName;
    private final String restaurantAddress;
    private final Double restaurantRating;


    public Restaurant(String restaurantName, String restaurantAddress, Double restaurantRating) {
        this.restaurantName = restaurantName;
        this.restaurantAddress = restaurantAddress;
        this.restaurantRating = restaurantRating;
    }

    public String getRestaurantName() {
        return restaurantName;
    }

    public String getRestaurantAddress() {
        return restaurantAddress;
    }

    public Double getRestaurantRating() {
        return restaurantRating;
    }
}

File: app/src/main/java/com/example/findmyrahmah/Restaurant.kt
package com.example.findmyrahmah

import android.media.Image
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Restaurant")

data class Restaurant(
    val image: Int,
    val name: String,
    val address: String,
    val rating: Float,
    var likeCount: Int,
    val isOnline: Boolean){

}
